dell_ai/models.py

```diff
@@ -17,19 +16,56 @@
 class ModelConfig(BaseModel):
     """Configuration details for a model deployment."""
 
+    model_config = {
+        "extra": "ignore",  # Ignore extra fields not defined in the model
+    }
+
+    backend: Optional[str] = None
+    model_id: Optional[str] = None
+    tensor_parallel_size: Optional[int] = None
```

Comment:
Can we add a TODO somewhere around this lines to flag that we should eventually update how the different environment variables or arguments per each engine is handled to prevent from having to manually capture all those and rather make this an arbitrary `Dict[str, Any]` with whatever the configuration is?